src/main/java/database/helper/Settings.java

```diff
@@ -19,6 +19,11 @@ public class Settings {
     private static final Logger LOG = LoggerFactory.getLogger(Settings.class);
     static final String DATABASE_STORAGE_PATH = "database.storage.path";
     static final String DEFAULT_DATABASE_STORAGE_PATH = "database";
+    public static final String LIMIT = "limit";
+    public static final String OFFSET = "offset";
+    public static final int DEFAULT_LIMIT_VALUE = 100;
+    public static final int DEFAULT_OFFSET_VALUE = 0;
```

Comment:
I think we don't need `offset` property at all. Please remove from code and `.properties` files.